CONTRIBUTING.md

```diff
@@ -1,42 +1,50 @@
 # Contributing to Catalyst
+
 Thanks for showing interest in contributing!
 
 The following is a set of guidelines for contributing to Catalyst. These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-#### Table of Contents
+## Repository Structure
+
+Catalyst is a monorepo that contains the code for the Catalyst Next.js application inside of `core/`, and supporting packages such as the GraphQL API client and the `create-catalyst` CLI in `packages/`.
+
+The default branch for this repository is called `canary`. This is the primary development branch where active development takes place, including the introduction of new features, bug fixes, and other changes before they are released in stable versions.
```

Comment:
Good opportunity to explain the tags and their practical usage (e.g., which Makeswift tag is deployed by OCC?, etc.)